be/src/olap/rowset/rowset.h

```diff
@@ -303,6 +303,9 @@ class Rowset : public std::enable_shared_from_this<Rowset> {
 
     std::string get_rowset_info_str();
 
+    virtual void clear_inverted_index_cache() { LOG(INFO) << "should not reach here"; }
```

Comment:
LOG(ERROR)?

Reply:
error is horrible for users.

Reply:
Move this method to `BetaRowset` private field